platform-core-module/src/main/java/com/softicar/platform/core/module/event/SystemEventPage.java

```diff
@@ -0,0 +1,39 @@
+package com.softicar.platform.core.module.event;
+
+import com.softicar.platform.core.module.CoreI18n;
+import com.softicar.platform.core.module.CoreModule;
+import com.softicar.platform.core.module.CoreRoles;
+import com.softicar.platform.core.module.module.instance.system.SystemModuleInstance;
+import com.softicar.platform.emf.authorization.role.IEmfRole;
+import com.softicar.platform.emf.management.page.AbstractEmfManagementPage;
+import com.softicar.platform.emf.page.EmfPagePath;
+import com.softicar.platform.emf.source.code.reference.point.EmfSourceCodeReferencePointUuid;
+import com.softicar.platform.emf.table.IEmfTable;
+
+@EmfSourceCodeReferencePointUuid("39723369-a6fd-4319-8149-2d71895386f3")
+public class SystemEventPage extends AbstractEmfManagementPage<SystemModuleInstance> {
+
+	@Override
+	public Class<CoreModule> getModuleClass() {
+
+		return CoreModule.class;
+	}
+
+	@Override
+	public EmfPagePath getPagePath(EmfPagePath modulePath) {
+
+		return modulePath.append(CoreI18n.MAINTENANCE);
```

Comment:
We should definitely rename `MAINTENANCE` to something more sensible.
Currently, theres a maintenance page, session overview page and after this PR the System event page.
It should definitely not stay with the name `MAINTENANCE`

Reply:
suggestions? I think the name is ok, tbh.

Reply:
I think we should ask the Ops dudes about this.

Reply:
did so, see screenshots